pom.xml

```diff
@@ -43,13 +47,11 @@
 
     <properties>
         <confluent.maven.repo>http://packages.confluent.io/maven/</confluent.maven.repo>
-        <confluent.common.version>3.2.1-SNAPSHOT</confluent.common.version>
-        <slf4j.version>1.7.21</slf4j.version>
+        <confluent.common.version>${project.version}</confluent.common.version>
```

Comment:
Should we just get rid of this variable entirely since it's the same as the project version anyway, or do you still see value in having the level of indirection? Perhaps to make any subsequent transition to using separate version numbers for different components in the future?

I'm mildly embarrassed that just reusing the project version didn't occur to me. We'd probably significantly simplify the version bumping tooling by doing this across the board, at least as long as we stick with fully coordinated version numbers.